components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -134,18 +87,183 @@ macro_rules! symbols {
                 #[cfg_attr(feature = "provider_serde", serde(borrow))]
                 pub stand_alone: Option<StandAloneWidthsV1<'data>>,
             }
-        };
-    }
+        }
+    };
+}
 
-symbols!(months, [Cow<'data, str>; 12]);
+symbols!(
+    months,
+    pub struct SymbolsV1<'data>(
+        #[cfg_attr(
+            feature = "provider_serde",
+            serde(borrow, with = "serde_with::As::<[serde_with::BorrowCow; 12]>")
+        )]
+        pub [Cow<'data, str>; 12],
```

Comment:
question: is this necessary for `[Cow; N]` as well?

Reply:
Yes